.changeset/nice-moose-clean.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/admin": major
```

Comment:
```suggestion
"@comet/admin": minor
```

That's not a breaking change

Reply:
@juliawegmayr can you change this to `minor` please?